File: CS/CustomCulture/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace CustomCulture {
    public partial class Form1 : Form {
        [STAThread]
        static void Main() {
            // set the modified culture globally
            //System.Globalization.CultureInfo culture;
            //culture = System.Threading.Thread.CurrentThread.CurrentCulture.Clone() as System.Globalization.CultureInfo;
            //culture.NumberFormat.NumberDecimalSeparator = ",";
            //culture.NumberFormat.NumberGroupSeparator = " ";
            //System.Threading.Thread.CurrentThread.CurrentCulture = culture;

            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new Form1());
        }

        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            textEdit1.Properties.Mask.MaskType = DevExpress.XtraEditors.Mask.MaskType.Numeric;
            textEdit1.Properties.Mask.UseMaskAsDisplayFormat = true;
            textEdit1.EditValue = 1234.56789m;
            textEdit2.Properties.Mask.MaskType = DevExpress.XtraEditors.Mask.MaskType.Numeric;
            textEdit2.Properties.Mask.UseMaskAsDisplayFormat = true;
            textEdit2.EditValue = 1234.56789m;

            //assign the modified culture only to a given editor
            System.Globalization.CultureInfo culture;
            culture = System.Threading.Thread.CurrentThread.CurrentCulture.Clone() as System.Globalization.CultureInfo;
            culture.NumberFormat.NumberDecimalSeparator = ",";
            culture.NumberFormat.NumberGroupSeparator = " ";
            textEdit1.Properties.Mask.Culture = culture;
        }
    }
}

File: VB/CustomCulture/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace CustomCulture
	Partial Public Class Form1
		Inherits Form
		<STAThread> _
		Shared Sub Main()
			' set the modified culture globally
			'System.Globalization.CultureInfo culture;
			'culture = System.Threading.Thread.CurrentThread.CurrentCulture.Clone() as System.Globalization.CultureInfo;
			'culture.NumberFormat.NumberDecimalSeparator = ",";
			'culture.NumberFormat.NumberGroupSeparator = " ";
			'System.Threading.Thread.CurrentThread.CurrentCulture = culture;

			Application.EnableVisualStyles()
			Application.SetCompatibleTextRenderingDefault(False)
			Application.Run(New Form1())
		End Sub

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			textEdit1.Properties.Mask.MaskType = DevExpress.XtraEditors.Mask.MaskType.Numeric
			textEdit1.Properties.Mask.UseMaskAsDisplayFormat = True
			textEdit1.EditValue = 1234.56789D
			textEdit2.Properties.Mask.MaskType = DevExpress.XtraEditors.Mask.MaskType.Numeric
			textEdit2.Properties.Mask.UseMaskAsDisplayFormat = True
			textEdit2.EditValue = 1234.56789D

			'assign the modified culture only to a given editor
			Dim culture As System.Globalization.CultureInfo
			culture = TryCast(System.Threading.Thread.CurrentThread.CurrentCulture.Clone(), System.Globalization.CultureInfo)
			culture.NumberFormat.NumberDecimalSeparator = ","
			culture.NumberFormat.NumberGroupSeparator = " "
			textEdit1.Properties.Mask.Culture = culture
		End Sub
	End Class
End Namespace